packages/compiler/core/diagnostics.ts

```diff
@@ -16,6 +16,7 @@ import {
   SourceLocation,
   SyntaxKind,
 } from "./types.js";
+import { NodeHost } from "./util.js";
```

Comment:
Yeah, i think this is the import that trigger that.

Reply:
Yep, just saw that. I reverted this file for now. I don't need this change. I'd like to make it, but it's completely separate. The change in this file was putting back diagnostic locations in the verbose test ouptu.

Reply:
I'll do this in a separate PR.

Reply:
```
```

Reply:
just adding `inlineDynamicImports: true,` under `output` solve the issue

Reply:
Yep, after I put in inlineDynamicImports it complained about a new lib without a license, though. So I'll punt on this. I should be able to avoid pulling in the extra deps and still make the change in this file, but the change in this file is not related to intellisense anyway, so I reverted it. 😁